unittests/backends/oqc/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+add_executable(test_oqc oqcTester.cpp)
```

I believe the filename case should be updated like this:
```suggestion
add_executable(test_oqc OQCTester.cpp)
```